### DISCUSSION TITLE
[PROPOSAL] Time-limited boosts

### DISCUSSION THREAD

**Discussion by brian-man** [sunflower-land/sunflower-land#2282] [General]
In part to combat "boost inflation", I think the game is overdue for introducing time-limited boosts.

Example 1:
2024 New Year's Glasses
Bumpkin Wearable
Quest in January 2024
Boost thru February 2024 (UTC): Lunar Calendar effect +10%

Example 2:
Fair Fox
Placeable
Unlimited supply in November 2023, 1 mint per land
Not withdrawable until 2024
Boost thru December 2023: Wheat yield +0.1

Example 3:
Fishing Lure
Bumpkin Wearable
Treasure Island, Rare, Seasonal
Boost for the 2 seasons: Fishing speed +25%

**Comment by Life-Is-Great** [comment]
I love this idea!
I just think that others would not like it in main gameplay but obtaining those boosts on treasure island, with magic beans and other side things/features would be awesome.

Good for economy and a good reward to get from new seasonal features.
